libs/application/templates/parental-leave/src/fields/Duration/index.tsx

```diff
@@ -21,7 +21,7 @@ import {
   getExpectedDateOfBirth,
 } from '../../parentalLeaveUtils'
 import { parentalLeaveFormMessages } from '../../lib/messages'
-import { usageMaxMonths, usageMinMonths } from '../../config'
+import { usageMaxMonths, usageMinMonths, daysInMonth } from '../../config'
```

Comment:
Looking at the values behind these constants

```typescript
export const usageMinMonths = 1
export const usageMaxMonths = 24
export const daysInMonth = 30
```

Shouldn't `useageMinMonths` be `0.5`?